tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -0,0 +1,172 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+"""This modules groups functions made to create reusable environments packaged in dagger containers."""
+
+from typing import List, Optional
+
+from ci_connector_ops.pipelines.contexts import ConnectorTestContext
+from ci_connector_ops.pipelines.utils import get_file_contents
+from dagger import CacheSharingMode, CacheVolume, Container, Directory, Secret
+
+PYPROJECT_TOML_FILE_PATH = "pyproject.toml"
+
+CONNECTOR_TESTING_REQUIREMENTS = [
```

Comment:
I'm not a huge fan of embedding these pins in a python file. Can we use a top level .env file or similar?

Reply:
I can create a `connector_testing_requirements.txt` file. I like to store it in a constant because it makes it easier to mock it for testing.